sourcegraph/sourcegraph
# JetBrains: fix onboarding bug with web flow

Previously, when the user authenticated with the web flow then you had to restart the IDE to let Cody pick up the new access token. This PR fixes the problem by sending an `extensionConfiguration/didChange` notification to the agent to inform it about the new credentials



## Test plan

- Remove all accounts
- Authenticate with dotcom web flow
- Send chat message, confirm that it responds
